vms/platformvm/state/state.go

```diff
@@ -1320,7 +1320,7 @@ func (s *state) syncGenesis(genesisBlk block.Block, genesis *genesis.Genesis) er
 
 	// Persist primary network validator set at genesis
 	for _, vdrTx := range genesis.Validators {
-		validatorTx, ok := vdrTx.Unsigned.(txs.ValidatorTx)
+		validatorTx, ok := vdrTx.Unsigned.(txs.ScheduledStaker)
```

Comment:
I don't think this is a problem - but previously we were enforcing that `vdrTx.Unsigned` was either an `AddValidatorTx` or an `AddPermissionlessValidatorTx`... Now a number of other txs would pass this check. It might make sense to comment the specific types we expect here or something.

Reply:
I added the comment and left a todo to go back to a stricter type check once Durango PRs are merged. The trick is to do it without duplicating the whole StakerTxs type hierarchy